packages/react/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
+
+export interface BadgeProps extends HTMLAttributes<HTMLElement> {
+  label: string | number
+  color?: 'blue' | 'dark-blue' | 'green' | 'dark-green' | 'orange' | 'magenta' | 'purple' | 'yellow' | 'black' | 'white'
```

Comment:
Now might also be a good time to create types for these.

Reply:
Perhaps when we start using primitives and semantic layers for colour tokens? Might be easier to get all supplemental colours that way.

Reply:
Yes, that will be great.

Created just a local variable for now, and derived the type and the tests from it.